app/Repositories/Eloquents/WebhookRepository.php

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace App\Repositories\Eloquents;
+ 
+use App\Models\Webhook;
+use App\Repositories\Interfaces\WebhookRepositoryInterface;
+use Auth;
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Classes.ClassDeclaration.OpenBraceNewLine</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/WebhookRepository.php</br>**Line**: 8</br>Opening brace of a class must be on the line after the definition